sdk/core/core-http/src/policies/bearerTokenAuthenticationPolicy.ts

```diff
@@ -15,6 +15,13 @@ import { WebResourceLike } from "../webResource";
 import { AccessTokenCache, ExpiringAccessTokenCache } from "../credentials/accessTokenCache";
 import { AccessTokenRefresher } from "../credentials/accessTokenRefresher";
 
+/**
```

Comment:
did this get auto-moved up from some kind of lint ordering rule? I'm fine with the change, just curious

Reply:
The rule that fired was `no-use-before-define`. In this case, `timeBetweenRefreshAttemptsInMs` was used in [bearerTokenAuthenticationPolicy](https://github.com/Azure/azure-sdk-for-js/blob/ffb7fe062715d3a8db3a38681eb1d41ea624c0c8/sdk/core/core-http/src/policies/bearerTokenAuthenticationPolicy.ts#L39). That's the only usage so I can move the constant inside the method if you prefer. Thanks for the review!

Reply:
That makes sense. I'm good with your solution!